_data/projects/nativescript.yml

```diff
@@ -0,0 +1,10 @@
+name: NativeScript
+desc: Cross platform Native mobile apps with JavaScript
+site: https://nativescript.org
+tags:
+- JavaScript
+- mobile
+- native
+upforgrabs:
+  name: PRs plz!
```

Comment:
generally we name this the same as the label on the repo - this was separated out so that we didn't need to do URL parsing on the client.